spec/controllers/ops_controller/settings/label_tag_mapping_spec.rb

```diff
@@ -30,7 +30,7 @@ def use_form_to_create_amazon_mapping
       expect(mapping.label_name).to eq('my-label')
       expect(mapping.label_value).to be nil
       expect(mapping.tag.classification.category?).to be true
-      expect(mapping.tag.classification.description).to eq('My Cat')
+      expect(mapping.tag.classification.description).to eq('kubernetes:container_project:|My Cat')
```

Comment:
The trailing `:` is strange to me ... Can that be removed?

Reply:
Yes, I can remove that 👍

Reply:
I think that no because `kubernetes:container_project:` is whole prefix from existing constant. Description with this prefix is not displayed in UI anywhere. Otherwise we would need to add additional logic for it.

Reply:
Sorry, missed your comment @lpichler. I played around with it locally and this change seems to work
```diff
+++ b/app/controllers/ops_controller/settings/label_tag_mapping.rb
@@ -251,7 +251,8 @@ module OpsController::Settings::LabelTagMapping
   end

   def cat_description_with_prefix(entity, cat_description)
-    "#{MAPPABLE_ENTITIES[entity].prefix}|#{cat_description}"
+    prefix = MAPPABLE_ENTITIES[entity].prefix[0..-2] # strip the trailing : char
+    "#{prefix}|#{cat_description}"
   end

   def cat_description_without_prefix(cat_description)
```
If you and @Fryguy are ok with it, I'll push that change

Reply:
@gtanzillo Prefer

```ruby
prefix = MAPPABLE_ENTITIES[entity].prefix.chomp(":")
"#{prefix}|#{cat_description}"
```